kolibri/core/assets/test/exams/utils.spec.js

```diff
@@ -447,14 +443,14 @@ describe('exam utils', () => {
           title: 'Count with small numbers',
         },
       ];
-      expect(converted).toEqual(
+      expect(converted.question_sources[0].questions).toEqual(
         expectedOutput.map(q => {
           q.item = `${q.exercise_id}:${q.question_id}`;
```

Comment:
I see the composite id we were using previously is being set as `item` on the questions by our functions here, is this not being used in the coach reports (i.e. why did we have to set it up as `id` separately?)

Reply:
The `item` property seems to be used in some parts of the front-end. I just kept it here rather than taking on the task of unifying them under one name right now.